app/Nova/Resources/Organization.php

```diff
@@ -53,22 +54,26 @@ public function fields(NovaRequest $request): array
                 ->rules('required', 'email', 'max:255'),
 
             Text::make('Vat')
+                ->hideFromIndex()
                 ->sortable()
                 ->rules('required', 'max:255'),
 
             Text::make('Street')
+                ->hideFromIndex()
                 ->sortable()
                 ->rules('required', 'max:255'),
 
-            Text::make('Zip')
+            Text::make('City')
                 ->sortable()
                 ->rules('required', 'max:255'),
 
-            Text::make('City')
+            Text::make('Zip')
+                ->filterable()
```

Comment:
Is this required? Because just typing `3000` in the search field will also work?

Reply:
I'll ask Corneel. An edge case might be if an `Organisation` has a name that contains like `1000Handen VZW` but has a zip of `2360` or `Tennis 2000` that has a zip of `9308`